releasenotes/notes/42400.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: traffic-management
+issue:
+- 42400
+releaseNotes:
+- |
+  **Fixed** updating service visibility does not push XDS to gateways
```

Nit: Add a period at the end of the sentence:
```suggestion
  **Fixed** updating service visibility does not push XDS to gateways.
```